app/views/candidate_mailer/_reasons_for_rejection_intro.text.erb

```diff
@@ -1,5 +1,9 @@
 <% unless @application_choice.rejected_by_default? %>
 
-  <%= @course.provider.name %> has decided not to make you an offer to study <%= @course.name_and_code %>. They've given feedback to explain this decision.
+  <%= @course.provider.name %> has reviewed your application for <%= @course.name_and_code %>.
+
+  On this occasion, they’ve decided not to make an offer.
+
+  They’ve given the following feedback:
```

Comment:
Do all the emails that use the `_reasons_for_rejection_intro` template give the feedback?